dockers/docker-fpm-frr/bgpcfgd

```diff
@@ -290,6 +291,24 @@ class BGPPeerMgr(Manager):
         vrf, nbr = key.split('|', 1)
         if key not in self.peers:
             cmd = None
+
+            if "local_addr" not in data:
+                syslog.syslog(syslog.LOG_ERR, 'Peer {}. Error in missing required attribute "local_addr"'.format(key))
+                return False
+
+            # Check if this route is belong to a vnet
+            interfaces = self.directory.get_slot("interface")
+            target_interface = None
+            for metadata in interfaces.values():
+                if metadata["ip"] == data["local_addr"]:
+                    target_interface = metadata
```

Comment:
should we break after we found match?

Reply:
Thanks for your suggestion. Have polished this flow.